torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -0,0 +1,205 @@
+"""Dispatcher for torch ATen decomposition and AtenLib functions from onnx-script."""
```

Comment:
```suggestion
"""Dispatcher for AtenLib functions from onnx-script."""
```
Not in this PR, decomposition table stuff should eventually be moved to fx frontend side.

Reply:
https://github.com/pytorch/pytorch/issues/94800